jest.global.setup.js

```diff
@@ -1,3 +1,3 @@
 module.exports = async () => {
-  process.env.TZ = 'UTC';
+  process.env.TZ = 'UTC'; // this does not affect Date objects on windows https://stackoverflow.com/a/56482581/210550
```

Comment:
We shouldn't have OS-specific code. If this doesn't work on Windows and it's not helpful in other operating systems, we should just remove it. Make sure to update Jest settings in `package.json` as well